### ISSUE TITLE
Selecting Combobox item by ENTER submits the form

### ISSUE THREAD

**Issue by adros** [jquense/react-widgets#290]
Chapter 'Keyboard navigation' in Combobox documentation says that Enter should be used to 'select focused item'. But this also submits the form in which the Combobox it. 

I suppose that the event should be stopped if it's case of selecting an item from drop down.

**Comment by jquense** [comment]
this was also in the batch of fixes I had stuck on a computer, released in 3.1.5
